src/app/main-api-handler.ts

```diff
@@ -93,8 +93,8 @@ ipcMain.on(apiName.symphonyApi, async (event: Electron.IpcMainEvent, arg: IApiAr
             downloadHandler.setWindow(event.sender);
             break;
         case apiCmds.showNotificationSettings:
-            if (typeof arg.windowName === 'string') {
-                windowHandler.createNotificationSettingsWindow(arg.windowName);
+            if (typeof arg.windowName === 'string' && typeof arg.theme === 'string') {
+                windowHandler.createNotificationSettingsWindow(arg.windowName, arg.theme);
```

Comment:
I think this would break in SFE 1.5 is that fine?

Reply:
That's a good catch, it's true, this'll break 1.5. I think you should catch the theme argument as optional and set a default @psjostrom